src/Frontend/Page/Idea.hs

```diff
@@ -287,33 +287,44 @@ instance ToHtml ViewIdea where
                     PhaseVoting{}     -> v >> c
                     PhaseResult       -> v >> c
 
-            div_ [class_ "sub-heading"] $ do
-                toHtml $ IdeaVoteLikeBars IdeaVoteLikeBarsWithButtons ctx stats
+            -- bars
+            toHtml $ IdeaVoteLikeBars stats
 
-            when (has _PhaseWildIdea phase && ideaReachedQuorum stats) $ do
-                div_ [class_ "table-actions m-no-hover"] $ do
-                    div_ [class_ "icon-list m-inline"] . ul_ $ do
+            -- indicators
+            div_ [class_ "table-actions m-no-hover"] $ do
+                div_ [class_ "icon-list m-inline"] . ul_ $ do
+                    when (has _PhaseWildIdea phase && ideaReachedQuorum stats) $ do
                         li_ [class_ "icon-table"] $ span_ "Kann auf den Tisch"
-                    when canCreateTopic $ do
-                        button_ [ class_ "btn-cta m-valid"
-                                , onclick_ $ U.Space spc U.CreateTopic
-                                ] $ do
-                            i_ [class_ "icon-check"] nil
-                            "Thema anlegen"
+                        feasibilityIndicator idea
 
-            feasibilityVerdict True idea caps
+            -- explanation by the dean why the idea is feasible or not (if available)
+            feasibilityVerdict idea
 
-            -- creator statement
-            when (any (`elem` caps) [CanAddCreatorStatement, CanEditCreatorStatement]) $ do
-                div_ [class_ "creator-statement-button"] $ do
+            -- buttons
+            toHtml $ ideaVoteLikeButtons ctx stats
+
+            when (has _PhaseWildIdea phase && ideaReachedQuorum stats && canCreateTopic) $ do
```

Comment:
you're right about the redundancy in the condition.  this kind of mistakes happens if you introduce too many names whose definitions are *almost* trivial, and no longer than the names themselves.  i've removed some of these names in this case.